docs/dev-environment/setting-up-dev-environment.md

```diff
@@ -1,44 +1,45 @@
 If you encounter unexpected errors during setup, or later during usage, see the `Frequently Encountered Errors` section at the bottom of this page.
 
-Also, if not obvious, all instructions in this setup guide assume that you have already cloned [this repo](https://github.com/thegazelle-ad/gazelle-server) which the wiki is located in, and that your current working directory is this repo.
+Also, if not obvious, all instructions in this setup guide assume that you have already cloned [this repo](https://github.com/thegazelle-ad/gazelle-server) which the repo is located in, and that your current working directory is this repo.
 
 # Install
 
 Make sure you have NodeJS and npm installed
 
-[Install nodejs and npm via nvm](https://github.com/creationix/nvm)
+You can [install nodejs and npm via nvm](https://github.com/creationix/nvm), feel free to install it any other way, but nvm is a nice way to manage your node versions if you work on different projects that use different versions.
 
-> **IMPORTANT**: For this, the main, repo we use **node v9.3.0** and **npm 5.6.0** (it's important not to use npm 5.5.x which may come as the default with node v9.3.0 as npm 5.5.x has a bug that breaks on node 9.3.0). Use any other versions at your own peril.
+> **IMPORTANT**: For this repo we use **node v9.11.1** and **npm 5.6.0**. Use any other versions at your own peril.
 
 Then install dependencies locally
 
 ```
 npm install
 ```
 
-# Connect to Ghost and Database
-To get Ghost and the database up and running on your server first clone [this repo](https://github.com/thegazelle-ad/database-and-ghost-blog-server) to your computer (outside the main repo) and follow the readme to get it set up.
+# Setup Database
+You should first install MariaDB, which can be done here: https://downloads.mariadb.org/mariadb/repositories. Simply follow the instructions to download MariaDB if you are using Linux. We use 10.1 in production / CircleCI so that's the recommended version. If you are not using Linux you should easily be able to Google how to download it for your operating system. If you are only setting up for development and not for deployment you can also use MySQL which is also 100% compatible.
```

Comment:
I would just add a sentence here about mac users: They can install it using homebrew, see this.

Change "If you are only setting up for development and not for deployment you can also use MySQL which is also 100% compatible." to "If you are only setting up for development and not for deployment you can also use MySQL which is 100% compatible." just to remove the extra also.